turn/__init__.py

```diff
@@ -2,7 +2,7 @@
 import requests
 import json
 
-from turn.exceptions import WhatsAppContactNotFound
+from turn.exceptions import *
```

Comment:
Instead of using a `*` import can you either import the exceptions you are catching, or instead:
`from turn import exceptions`